continuationsPlugin/src/main/scala/continuations/Suspend.scala

```diff
@@ -1,9 +1,22 @@
 package continuations
 
+import continuations.Suspend.CompilerRewriteUnsuccessfulException
+
 sealed trait Suspend:
   def continuation[A](f: Continuation[A] => Unit): A
 
+  extension (s: Suspend)
+    inline def suspendContinuation[A](f: Continuation[A] => Unit): A =
+      throw CompilerRewriteUnsuccessfulException
+
 object Suspend:
+  case object CompilerRewriteUnsuccessfulException
```

Comment:
```suggestion
  private[continuations] case object CompilerRewriteUnsuccessfulException
```
We don't want anyone else calling this.